test/test_ssl_service.py

```diff
@@ -0,0 +1,92 @@
+import io
+import unittest
+from contextlib import redirect_stderr
+from types import SimpleNamespace
+
+from conjur.ssl_service import SSLService
+
+GITHUB_FINGERPRINT = "5F:3F:7A:C2:56:9F:50:A4:66:76:47:C6:A1:8C:A0:07:AA:ED:BB:8E"
+GITHUB_CERT = \
```

Comment:
Its a secrat that shouldn't be here on git not ??

Reply:
Or its just for the tests and its ojay?

Reply:
That's the public client cert for GH when you try to open a a socket and get make a request to get it's cert chain. Didn't need any special authorization to access this info so it's forward facing. This is actually how we currently test in our Ruby CLI tests

Reply:
Wonderful test, and great reuse of code from RUBY CLI!
Hugely simplifies the testing of this code section, isn't it?!